quiche/src/recovery/mod.rs

```diff
@@ -566,6 +569,50 @@ impl ReleaseDecision {
     }
 }
 
+/// Recovery statistics
+#[derive(Default, Debug)]
+pub struct RecoveryStats {
+    startup_exit: Option<StartupExit>,
+}
+
+impl RecoveryStats {
+    // Record statistics when a CCA first exits startup.
+    pub fn set_startup_exit(&mut self, startup_exit: StartupExit) {
+        if self.startup_exit.is_none() {
+            self.startup_exit = Some(startup_exit);
+        }
+    }
+}
+
+/// Statistics from when a CCA first exited the startup phase.
+#[derive(Debug, Clone, Copy, PartialEq)]
+pub struct StartupExit {
+    /// The congestion_window recorded at Startup exit.
+    pub cwnd: usize,
+
+    /// The reason a CCA exited the startup phase.
+    pub reason: StartupExitReason,
+}
+
+impl StartupExit {
+    fn new(cwnd: usize, reason: StartupExitReason) -> Self {
```

Comment:
I think this function is now unused after your refactor to [startup.rs](https://github.com/cloudflare/quiche/pull/2064/files#diff-0d91abc11d379b124d2a6a20e11d1e04bc52581bbedffcc9a3cad414ec45b5e8)